docs-new/docs/graphql/core/api-reference/relay-graphql-api/query.mdx

```diff
@@ -170,46 +152,39 @@ type tableEdge {
 }
 ```
 
-### Cursor
+### Cursor {#cursor}
 
 The cursor field returns a unique non-null String value which is useful
-for `pagination <RelayPaginationExp>`.
-
-<div class="note">
-
-<div class="title">
+for [pagination](#paginationexp).
 
-Note
+:::info Note
 
-</div>
-
-For more details on the Relay `cursor`, refer to the [Relay
-docs](https://relay.dev/graphql/connections.htm#sec-Cursor).
+For more details on the Relay `cursor`, refer to the [Relay docs](https://relay.dev/graphql/connections.htm#sec-Cursor).
 
-</div>
+:::
 
 ### Argument
 
 <div class="parsed-literal">
 
-`DistinctOnExp <RelayDistinctOnExp>` \| `WhereExp <RelayWhereExp>` \|
-`OrderByExp <RelayOrderByExp>` \| `PaginationExp <RelayPaginationExp>`
+[DistinctOnExp](#distinctonexp) | [WhereExp](#whereexp) |
+[OrderByExp](#orderbyexp) | [PaginationExp](#paginationexp)
```

Comment:
broken ids - relay prefix missed

please correct id for all 4 sub-section links by referring their custom ids

Reply:
fixed